tox.ini

```diff
@@ -47,6 +47,9 @@ commands=
 
 [testenv:linting]
 basepython = python2.7
+# needed to keep check-manifest working
+setenv =
+    SETUPTOOLS_SCM_PRETEND_VERSION=2.0.1
```

Comment:
So it seems this fixes the check-manifest issue. 😁 

Can we switch to the `write_to` option now?